kf_release_maker/release_maker.py

```diff
@@ -1,276 +1,198 @@
         """
-        if version_type not in config.VALID_VERSION_TYPES:
+        parts = [int(p) for p in prev_version.split(".")]
+        if release_type == MAJOR:
+            new_version = f"{parts[0]+1}.0.0"
+        elif release_type == MINOR:
+            new_version = f"{parts[0]}.{parts[1]+1}.0"
+        elif release_type == PATCH:
+            new_version = f"{parts[0]}.{parts[1]}.{parts[2]+1}"
+        else:
             raise ValueError(
-                f'Invalid version type: {version_type}! Version type '
-                f'must be one of {pformat(config.VALID_VERSION_TYPES)}!'
+                f"Invalid release type: {release_type}! Release type "
+                f"must be one of ['{MAJOR}', '{MINOR}', '{PATCH}']!"
             )
 
-        parts = [int(p) for p in prev_version.split('.')]
-        if version_type == 'major':
-            new_version = f'{parts[0]+1}.0.0'
-        elif version_type == 'minor':
-            new_version = f'{parts[0]}.{parts[1]+1}.0'
-        else:
-            new_version = f'{parts[0]}.{parts[1]}.{parts[2]+1}'
-
-        print(f'Next release version is {new_version}')
-
         return new_version
 
-    def create_release_markdown(self, pr_df):
-        """
-        Create release note markdown from PR DataFrame and
-        Write to a markdown file named: `<repo name>-<release version>.md`
-        """
-        print('Creating release note markdown ...\n')
-
-        # Emoji count summary markdown string
-        emoji_counts = ' '.join([
-            f'{idx} x{row["number"]}'
-            for idx, row in pr_df.groupby('emoji').count().iterrows()
-        ])
-        # Category count summary markdown string
-        endpoint = f'{self.base_url}/labels'
-        category_counts = ' '.join([
-            f'[{cat}]({endpoint}/{cat}) x{row["number"]}'
-            for cat, row in pr_df.groupby('category').count().iterrows()
-        ])
-        # PR summary markdown string
-        prs = '\n'.join(
-            [f'- (#{row["number"]}) {row["title"]} - @{row["user"]}'
-             for _, row in pr_df.iterrows()]
-        )
-
-        # Construct release notes markdown
-        title = (
-            'Kids First' + ' '.join(self.repo.split('-')).title().lstrip('Kf')
-        )
-        notes = f"# {title} Release {self.release_version}"
-        notes += '\n\n## Features\n\n'
-        notes += '### Summary\n\n'
-        notes += f'Feature Emojis: {emoji_counts} \n'
-        notes += f'Feature Categories: {category_counts} \n\n'
-        notes += '### New features and changes\n\n'
-        notes += prs
-
-        print('#'*80)
-        print(notes+'\n\n')
-
-        # Write markdown file
-        file_name = f'{self.repo}-{self.release_version}.md'
-        with open(file_name, 'w') as f:
-            f.write(notes)
+    def _to_markdown(self, repo, version, counts, prs):
+        messages = [
+            f"Release {version}",
```

Comment:
```suggestion
            f"# Release {version}",
```

Reply:
Can you also prepend the title cased version of the org and repo name (e.g. Kids First Kf Data Lib Ingest)

Reply:
"Kf Data Lib Ingest" looks bad to me

Reply:
See https://github.com/kids-first/kf-release-maker/pull/9/commits/beeb21ab570d1e03cb6d8347dfe0bbcc8d47fbe7